precious.toml

```diff
@@ -26,3 +26,41 @@ cmd = [
 ]
 ok_exit_codes = [0]
 lint_failure_exit_codes = [1]
+
+[commands.prettier-yaml]
+type = "both"
+invoke = "once"
+working-dir = "root"
+path-args = "file"
+include = [ "**/*.yml", "**/*.yaml" ]
+cmd = "$PRECIOUS_ROOT/node_modules/.bin/prettier"
+lint-flags = "--check"
+tidy-flags = "--write"
+ok-exit-codes = [0]
+lint-failure-exit-codes = [1]
+ignore-stderr = ".*"
+
+[commands.prettier-md]
+type = "both"
+invoke = "once"
+working-dir = "root"
+path-args = "file"
+include = [ "*.md" ]
+exclude = [
+    ".github/pull_request_template.md",
+    ".github/top-comment-content.md",
+]
```

Comment:
(nit) You don't need these here; copy-pasted from elsewhere.

Reply:
Fixed.